### ISSUE TITLE
Stalls when after restarting computer or first turning on, on Win10

### ISSUE THREAD

**Issue by rednaxela1986** [Belphemur/SoundSwitch#239]
Do not have an error message but when first starting Windows hotkeys are not working. Right-click it in sys tray and will not let me click on options or anything else. Have to end process and restart the program to get it to work.

**Comment by 808pants** [comment]
Me too - only I don't see anything running in processes that looks like SoundSwitch, so I can't halt anything and try again.  
'Soundswitch' icon remains faithfully in my taskbar (when it works, it works great) but has never responded quite as I'd expect, and only works about half the time.  

At present - right after a Win10 update:
left click has no apparent effect
two right-clicks in sequence (that's what it takes to get into the app?) offer options of 'open' and 'run as admin'.  Of those options, 'open' has no effect, but 'run as admin' begins a UAC verification - then goes silent, so... who knows.

**Comment by Belphemur** [comment]
Can you try a different type of notifcation than Banner ?

**Comment by 808pants** [comment]
> ...different type of notifcation than Banner ?

who, me?  Explain, please...I don't know what that means...can't Google an obvious answer, either.

**Comment by Belphemur** [comment]
Both of you :)

In the settings of SoundSwitch you can choose what type of notification to use. By default it's the banner type. I'm asking you to try another type of notification like Windows Default or Windows Toast notification.

**Comment by 808pants** [comment]
Antoine,

I can't open SoundSwitch - at all - so I don't see how I can change
settings.  I found my PC freshly rebooted last night when I got home, too,
so the problem persists after reboot.

In the process of messing around with this, I only recently discovered that
it's relatively easy to change outputs using the built-in speaker,
um...banner?  (click on speaker icon in tray, then up-arrow to make
selection)  It's not as slick/quick as your hotkey control, but not too bad.

Dave
